ClientApp/viking-vault/src/UserDashboard/TransactionList.css

```diff
@@ -0,0 +1,19 @@
+.transactions {
+    width: 50%;
+    position: absolute;
+    right: 5%;
+    bottom: 5%;
+}
+
+.transactions table {
+    color: #58595B;
+}
+
+.transactions h5 {
+    color: #004450;
+}
+
+.transactions img {
```

Comment:
It's better to avoid using elements names as much as possible and instead to use classes, so if the image represent an user image instead of saying `.transactions img`, you should rather say `.transactions .user-image`